docs/xrefs.json

```diff
@@ -19,6 +19,58 @@
     "title": "Guarantees",
```

Comment:
Don't commit this file

Reply:
Don't we want it to always be up to date?

Reply:
Ah, I see. #935 should take care of that.